sonarts/Dockerfile

```diff
@@ -0,0 +1,65 @@
+FROM debian:8.7
```

Comment:
Why don't use alpine?

Reply:
I wanted to be the closest to docker ekino's node image, so I use the same origin and use the already working scripts, but maybe a bad idea ?
Is there a global consensus of usage of alpine on all ekino's docker images ?

Reply:
You're right, I have some doubt at the beginning, but I can do almost the same that it was done for java via a docker template file to allow multiple node's version.